src/drive/web/modules/move/MoveModal.jsx

```diff
@@ -89,10 +97,39 @@ class MoveModal extends React.Component {
     }
   }
 
-  moveEntry = (entryId, destinationId) => {
-    return this.context.client
-      .collection('io.cozy.files')
-      .updateFileMetadata(entryId, { dir_id: destinationId })
+  /**
+   * Try to move entry to destination.
+   *
+   * @param   {string} entryId       - The file's id
+   * @param   {string} destinationId - The target directory's id
+   * @param   {string} filename      - The file's name
+   * @returns {Promise}              - A promise that returns the move action response if resolved or an Error if rejected
+   *
+   */
+  moveEntry = async (entryId, destinationId, filename) => {
+    const { client } = this.context
+    try {
+      return await client
+        .collection('io.cozy.files')
+        .updateFileMetadata(entryId, { dir_id: destinationId })
```

Comment:
I don't think we need to await in this method (except when we have to wait a result before going on).